### DISCUSSION TITLE
【質問欄】もっと強制力のあるポリシーが必要だと思います

### DISCUSSION THREAD

**Discussion by Verclene** [increments/qiita-discussions#917] [要望・アイデア・不具合]
説明するのもバカバカしいのでこの辺の惨状とかを[ご覧いただきたいのですが(ご指摘によりリンク削除しました)
テンプレを守らない質問者も煽りやマナー自治厨しかしない回答者もそれなりに増えて見苦しいので,いい加減運営が強制力を以て対処してほしいと感じます.
治安という一言で片付くものでもありませんが,質問欄がナレッジベースとしては現状あまりにも役に立たなさすぎるので何らかアクションが必要なのではないでしょうか.

**Comment by tanaka-asami415** [comment]
@Verclene 
ご意見ありがとうございます 🙏

現在、コミュニティガイドラインに沿って対応を行っておりますが、現状の対応だけでは解決できていない事象もある事は運営としても課題に感じております。

今後の方向性としては、更に運営の強制を強めるのではなく、仕組みでユーザー行動に影響を与えるような改善をしていきたいと考えております。
（例えば、コミュニティから好ましくないと判断された方へのペナルティ等の仕組みの強化や、ガイドラインをより意識してもらえるような改善など）

厳しくしすぎる事によって、投稿しづらい雰囲気を作ってしまう懸念もあるため、様子を見ながら時間をかけての改善になると思います。ご理解いただけると幸いです。

また、例として挙げていただいたものではありますが、特定のアカウントを指摘する行為になりますので、お控えいただけますと幸いです🙇
